eth/backend.go

```diff
@@ -233,6 +233,8 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 		overrides.OverrideOptimismInterop = config.OverrideOptimismInterop
 	}
 	overrides.ApplySuperchainUpgrades = config.ApplySuperchainUpgrades
+	overrides.EnableSoulGasToken = config.EnableSoulGasToken
```

Comment:
I guess we also need to remove these two lines?

Reply:
Fixed.